src/puppetlabs/puppetdb/query_eng/engine.clj

```diff
@@ -49,7 +48,7 @@
   [column-keyword]
   (->> column-keyword
        name
-       su/sql-uuid-as-str
+       (#(str % "::text"))
```

Comment:
It would be great if this had a name, would be more clear what appending ::text does